__tests__/src/component/common/MultiSelect.js

```diff
@@ -0,0 +1,47 @@
+import MultiSelect from '../../../../src/components/MultiSelect';
+import React from 'react';
+import renderer from 'react-test-renderer'
+import Colors from "../../../../src/global/Colors";
+import Strings from '../../../../src/global/Strings'
+
+test('MultiSelect renders correctly', () => {
+  const tree = renderer.create(
+    <MultiSelect
+      hideTags
+      items={[{
+        id: 'id1',
+        name: 'Testing Opt 1',
+      }, {
+        id: 'id2',
+        name: 'Testing Opt 2',
+      }]}
+      uniqueKey="id"
+      submitButtonText={Strings.ok}
+    />
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+test('MultiSelect renders correctly with customized colors', () => {
+  const tree = renderer.create(
+    <MultiSelect
+      hideTags
+      items={[{
+        id: 'id1',
+        name: 'Testing Opt 1',
+      }, {
+        id: 'id2',
+        name: 'Testing Opt 2',
+      }]}
+      uniqueKey="id"
```

Comment:
Please do not mix double and single quotes (optional)

Reply:
Corrected